CI failure in python 3 unit tests:

```
>       return _compile(pattern, flags).search(string)
E       TypeError: cannot use a string pattern on a bytes-like object

/usr/lib/python3.5/re.py:173: TypeError
```

--- a/test/units/module_utils/facts/hardware/linux_data.py
+++ b/test/units/module_utils/facts/hardware/linux_data.py
@@ -49,7 +49,7 @@ LSBLK_UUIDS = {'/dev/sda1': '66Ojcd-ULtu-1cZa-Tywo-mx0d-RF4O-ysA9jK'}
 
 UDEVADM_UUID = 'N/A'
 
-UDEVADM_OUTPUT = b"""
+UDEVADM_OUTPUT = """
 UDEV_LOG=3
 DEVPATH=/devices/pci0000:00/0000:00:07.0/virtio2/block/vda/vda1
 MAJOR=252

--- a/test/units/module_utils/facts/hardware/test_linux.py
+++ b/test/units/module_utils/facts/hardware/test_linux.py
@@ -171,4 +171,4 @@ class TestFactsLinuxHardwareGetMountFacts(unittest.TestCase):
         lh = linux.LinuxHardware(module=module, load_on_init=False)
         udevadm_uuid = lh._udevadm_uuid('mock_device')
 
-        self.assertEqual(udevadm_uuid, b'57b1a3e7-9019-4747-9809-7ec52bba9179')
+        self.assertEqual(udevadm_uuid, '57b1a3e7-9019-4747-9809-7ec52bba9179')

